spring-security-sso/spring-security-sso-kerberos/src/main/java/kerberos/client/SampleService.java

```diff
@@ -0,0 +1,24 @@
+package kerberos.client;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+class SampleService {
+
+	@Value("${app.access-url}")
+	private String endpoint;
+
+	private RestTemplate restTemplate;
+
+	@Autowired
+	public void setRestTemplate(RestTemplate restTemplate) {
```

Comment:
Use constructor-based injection instead? :)

Reply:
thx - adjusted. Will leave setter for the sake of test showcase.